.github/workflows/ci.yml

```diff
@@ -155,8 +154,8 @@ jobs:
           flags: unittest
           name: codecov
 
-  linting_and_style:
-    name: Code style and lint
+  spellcheck:
```

Comment:
Split this out just because i was annoyed when one failed and it killed the entire lint job.

Reply:
+1 from me on the splitup!

...and continued irk at GHA for not making it easy to reuse steps copypasta...